content/en/user-guide/aws/s3/index.md

```diff
@@ -159,11 +221,34 @@ However, if you try to access your bucket from [LocalStack Web Application](http
 }
 ```
 
-You can now run the same steps as before to update the CORS configuration and verifying if it is applied correctly:
+You can now run the same steps as before to update the CORS configuration and verify if it is applied correctly:
 
 {{< command >}}
 $ awslocal s3api put-bucket-cors --bucket cors-bucket --cors-configuration file://cors-config.json
 $ awslocal s3api get-bucket-cors --bucket cors-bucket
 {{< / command >}}
 
 You can try again to upload files in your bucket from the [LocalStack Web Application](https://app.localstack.cloud) and it should work.
+
+## Resource Browser
+
+The LocalStack Web Application provides a [Resource Browser](https://docs.localstack.cloud/user-guide/web-application/resource-browser/) for managing S3 buckets & configurations. You can access the Resource Browser by opening the LocalStack Web Application in your browser, navigating to the **Resources** section, and then clicking on **S3** under the **Storage** section.
+
+<img src="s3-resource-browser.png" alt="S3 Resource Browser" title="S3 Resource Browser" width="900" />
+
+The Resource Browser allows you to perform the following actions:
+
+- **Create Bucket**: Create a new S3 bucket by specifying a **Bucket Name**, **Bucket Configuration**, **ACL**, **Object Ownership**, and more.
+- **Objects & Permissions**: View, upload, download, and delete objects in your S3 buckets. You can also view and edit the permissions, like the CORS Configuration, for each object.
```

Comment:
the CORS configuration is bucket-wide and not for each objects. I guess we would need something related to bucket configuration in this list?